### ISSUE TITLE
Document "Host and Pool have incompatible Licenses" error

### ISSUE THREAD

**Issue by olivierlambert** [xcp-ng/xcp#509]
This might happen when you try to add XCP-ng hosts in a "mixed" pool (eg master on XCP-ng and slaves on Citrix Hypervisor).

We should document the fact that you must get your pool "coherent" (with all XCP-ng hosts at the same version) before trying to add new hosts in it.

**Comment by Fohdeesha** [comment]
To clarify, this issue doesn't happen when adding an existing mixed pool to XOA via Settings > Servers right? It only happens when you have a mixed pool added to XOA, then you try to add a new slave to the pool? Which doc would be the best fit for it? Perhaps https://xcp-ng.org/docs/upgrade.html#upgrade-from-xenserver ?

**Comment by olivierlambert** [comment]
Correct: not related to XOA. It's just when you have a mixed pool (ie inside your pool, various versions) and you want to add a new member to that pool.

XOA can connect to any number of pool regardless then version (XCP or XenServer).

**Comment by Fohdeesha** [comment]
Going to push this via PR soon, should we put it under:  

https://xcp-ng.org/docs/upgrade.html#upgrade-from-xenserver

or: 

https://xcp-ng.org/docs/troubleshooting.html

**Comment by stormi** [comment]
Maybe both: a small sentence in upgrade.md telling users not to mix XS with XCP-ng, and a more detailed troubleshooting section?

**Comment by Fohdeesha** [comment]
submitted via https://github.com/xcp-ng/xcp-ng-org/pull/138
